### PR TITLE
Updating Makefile to reflect current structure of repository

### PR DESCRIPTION
**Pull request by rawkode**
Signed-off-by: David McKay <david.andrew.mckay@gmail.com>

## Description
- Removed SQUASH option, as Dockerfile is now multi-stage
- Fixed installation targets to use YAML directories

## Motivation and Context

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

Makefile targets were broken and SQUASH leads to failed builds when `make build` and not enabled experimental daemon

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -16,19 +15,16 @@ build-armhf:
...
 
 push:
 	docker push alexellis2/faas-netes:$(TAG)
 
 install:
-	kubectl apply -f faas.yml,monitoring.yml,rbac.yml
-
-install-async:
-	kubectl apply -f faas.async.yml,monitoring.yml,rbac.yml,nats.yml
+	kubectl apply -f yaml/
```
Comment (alexellis) [inline Makefile]: Will also need to apply ./namespaces.yml

File: Makefile [hunk 2]
```diff
@@ -16,19 +15,16 @@ build-armhf:
 	docker build -t functions/faas-netesd:$(TAG)-armhf . -f Dockerfile.armhf
 
 build:
-	docker build --build-arg http_proxy="${http_proxy}" --build-arg https_proxy="${https_proxy}" -t functions/faas-netesd:$(TAG) . --squash=${SQUASH}
+	docker build --build-arg http_proxy="${http_proxy}" --build-arg https_proxy="${https_proxy}" -t functions/faas-netesd:$(TAG) .
```
Comment (alexellis) [inline Makefile]: Double check, but I think the image name is now `openfaas/` as a prefix

File: Makefile [hunk 3]
```diff
@@ -1,5 +1,4 @@
 TAG?=latest
-SQUASH?=false
```
Comment (alexellis) [inline Makefile]: 👍

**Comment by rawkode** [comment]
@alexellis Should be good :+1:

**Comment by alexellis** [comment]
LGTM, thank you for this @rawkode

**Pull request merged**